pkg/scheduler/core/generic_scheduler.go

```diff
@@ -806,9 +806,9 @@ func PrioritizeNodes(
 		}
 	}
 
-	for _, scoreList := range scoresMap {
-		for i := range nodes {
-			result[i].Score += scoreList[i]
+	for _, nodeScoreList := range scoresMap {
```

Comment:
Can this be merged into previous `for i := range nodes` loop, after L806?

Reply:
Good catch, Done